README.md

```diff
@@ -301,6 +301,24 @@ Notes are sorted by "relevance". Currently it means the notes that contain the r
 
 In the desktop application, press Ctrl+G or Cmd+G and type the title of a note to jump directly to it. You can also type `#` followed by a tag or `@` followed by a notebook title.
 
+# Privacy
+
+Joplin values your privacy and security by giving you complete control over your information and digital footprint.
+
+Joplin applications do not send any data to any service without your authorisation. Any data that Joplin saves, such as notes or images, are saved to your own device and you are free to delete this data at any time.
+
+Joplin has many modern features, some of which use third-party services. You can be completely disable any or all of these features in the application settings. These features are:
+
+Feature | Description | Default
+--------|-------------|---------
+|Auto-update|Joplin periodically connects to github to check for new releases.|Enabled|
+|Geo-location|Joplin saves geo-location information in note properties when you create a note.|Enabled.|
+|Synchronisation|Joplin supports synchronisation of your notes across multiple devices. If you choose to synchronise with a third-party, such as OneDrive, the notes will be sent to your OneDrive account, in which case the third-party privacy policy applies. |Enabled, Dropbox, 5 minute intervals|
```

Comment:
The user needs to login to Dropbox first so it's not really enabled by default. I don't believe the app even pings Dropbox till the user is logged in.

Reply:
I suspected this, but thought it was better to err on the side of caution.

In the options, `Dropbox` is selected by default with a `5 minute` interval, so I thought it would cause confusion if someone was to read it is disabled while it looks like it is enabled in the settings.  Perhaps we should have a default `Disabled` option in the setting?

Also note it says the following in the Synchronisation section higher up in the `README.md`:
> Currently, synchronisation is possible with Nextcloud, Dropbox (by default), OneDrive or the local filesystem. 

I'll need to check the pings are disabled.

Reply:
> Perhaps we should have a default `Disabled` option in the setting?

Actually, I see why it is that way now.  `Disabled` would be clearer, but less "nice".  I'll just update the text here to say it is disabled.

Reply:
I've fixed it to say synchronisation is disabled by default.

To try to reduce the chance of any confusion, I also made two minor edits to the **Synchronisation** section in `README.md`

Reply:
Confirmed by experiment it is not pinging Dropbox, so truly disabled.